tests/test_autoscaling_lib.py

```diff
@@ -922,10 +923,11 @@ def test_set_spot_fleet_request_capacity():
         mock.patch('paasta_tools.autoscaling_lib.AWS_SPOT_MODIFY_TIMEOUT', autospec=True)
```

Comment:
It is testing a bit more than just the timeout. Also I'm a fan of the `test_<method_name>` format. Let me know if you feel strongly but otherwise I'll leave it as is.